helpers/value_in_range

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+. bootrr
+
+TEST_CASE_ID="$1"
+LOC="$2"
+LOW="$3"
+HIGH="$4"
+TIMEOUT="${5:-1}"
```

Comment:
unused.

Reply:
well, I don't understand how bjorn uses TIMEOUT in the other helpers, in fact.. I don't grasp what
timeout 10 [ -L ${PARTITION} ] || test_report_exit blocked
is doing.. 

if it does anything useful, then you might want to add timeout as well below..